src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBinding.cs

```diff
@@ -25,7 +25,9 @@ public FunctionBinding(FunctionDescriptor descriptor, IReadOnlyDictionary<string
         {
             Dictionary<string, IValueProvider> results = new Dictionary<string, IValueProvider>();
 
-            BindingContext bindingContext = new BindingContext(context, null);
+            // Supplied bindings can be direct parameters or route parameters. 
```

Comment:
Correct - this won't solve trigger functions. 

---
In reply to: [62585581](https://github.com/Azure/azure-webjobs-sdk/pull/719#discussion_r62585581) [](ancestors = 62585581)